### PR TITLE
Models for topic and sentiment analysis

### PR DESCRIPTION
**Pull request by jgibson517**
@abejburton opening this draft for your awareness. topic and sentiment analysis functions are good to go, will start integrating into the ml pipeline // writing tests

### CODE REVIEW

**Comment by abejburton** [comment]
I'm making progress on the pipeline, will try to get something simple running today then push a draft pr so we can then make sure our two branches are integrating well

**Comment by jgibson517** [comment]
@abejburton I pushed a commit with a draft for a`Comments` dataclass if you want to take a look

**Review by abejburton — CHANGES_REQUESTED** [review]
Want to discuss timestamps and make sure we aren't creating conflicts with the pipeline code that will be merged into main first. Other than that, looks great and ready to go

File: civiclens/nlp/topics.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+class Comments(BaseModel, arbitrary_types_allowed=True):
+    df: pl.DataFrame
+    document: str = ""
+    rep_comments: list[str] = []
+    num_comments: int = 0
+    num_unique_comments: int = 0
+    num_rep_comments: int = 0
+    topics: dict = {}
+    created_at: datetime = datetime.today()
```
[the remark this answers is not in the document]
Comment (jgibson517) [reply civiclens/nlp/topics.py:25]: sounds good! I'm going to remove this class and integrate any other features onto your `DocComments` class

**Comment by jgibson517** [comment]
@abejburton made a few tweaks but this is now go to go, lets merge this and your pipeline draft and then i'll open a PR to integrate//complete the pipeline

**Comment by abejburton** [comment]
@jgibson517 looks good, just submitted all my revisions so you can move ahead with that

**Review by abejburton — APPROVED** [review]

**Pull request merged**